bobtemplates/plone_addon/src/+package.namespace+/+package.name+/profiles/default/types/+package.dexterity_type_name+.xml.bob

```diff
@@ -1,44 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<object name="{{{ package.dexterity_type_name }}}" meta_type="Dexterity FTI" i18n:domain="{{{ package.dottedname }}}"
-   xmlns:i18n="http://xml.zope.org/namespaces/i18n">
-  <property name="title" i18n:translate="">{{{ package.dexterity_type_name }}}</property>
-  <property name="description"
-    i18n:translate=""></property>
-  <property name="icon_expr"></property>
+<object
+    i18n:domain="{{{ package.dottedname }}}"
+    meta_type="Dexterity FTI"
+    name="{{{ package.dexterity_type_name }}}"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">{{{ package.dexterity_type_name }}}</property>
```

Comment:
and if you explain somebody a file its great to tell: "look at line no.4 we have the ..." and so on.